docs/command_line_tool/phylum_auth_revoke-token.md

```diff
@@ -0,0 +1,27 @@
+---
+title: phylum auth revoke-token
+category: 6255e67693d5200013b1fa3e
+hidden: false
+---
+
+Revoke an API token
+
+```sh
+Usage: phylum auth revoke-token [OPTIONS] [token-name]...
+```
+
+### Arguments
+
+
+&emsp; Unique token names which identify the tokens
```

Comment:
```suggestion
[token-name]
&emsp; Unique token names which identify the tokens
```

This doc is auto-generated so the missing entry here is likely indicative of a problem in the `clap_markdown` crate, but the expectation is that the format here would look like other commands with a `[blah]...` argument (e.g., `docs/command_line_tool/phylum_parse.md`)